### PR TITLE
Workspaceify auto

### PR DESCRIPTION
**Pull request by welbon**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,209 +1,7 @@
...
...xecutor", "executor/benchmark", "contrib-contracts", "chain", "chain/api", "chain/open-block", "chain/mock", "chain/chain-notify", "chain/service", "node/api", "node", "sync", "sync/api", "block-relayer", "miner", "node", "network-p2p", "network-p2p/types", "network-p2p/peerset", "network", "network/types", "network/api", "network-rpc", "network-rpc/derive", "network-rpc/core", "network-rpc/api", "account/api", "account", "account/service", "account/provider", "rpc/api", "rpc/middleware", "rpc/client", "rpc/server", "vm/types", "vm/vm-runtime", "vm/natives", "vm/stdlib", "vm/compiler", "vm/move-prover", "vm/transaction-builder", "vm/transaction-builder-generator", "vm/move-coverage", "vm/resource-viewer", "vm/dev", "vm/move-explain", "vm/move-package-manager", "vm/vm-status-translator", "vm/starcoin-transactional-test-harness", "abi/types", "abi/decoder", "abi/resolver", "test-helper", "cmd/starcoin", "cmd/faucet", "cmd/tx-factory", "cmd/miner_client", "cmd/generator", "dataformat-generator", "cmd/indexer", "cmd/resource-exporter", "cmd/merkle-generator", "cmd/peer-watcher", "cmd/airdrop", "cmd/replay", "stratum", "cmd/miner_client/api", "cmd/db-exporter", "cmd/genesis-nft-miner",]
```
Comment (nkysg) [inline Cargo.toml:3]: 这里格式还是按照原来按行来吧，感觉更好阅读

File: Cargo.toml [hunk 2]
```diff
@@ -1,209 +1,7 @@
...
...xpool/api", "txpool/mock-service", "executor", "executor/benchmark", "contrib-contracts", "chain", "chain/api", "chain/open-block", "chain/mock", "chain/chain-notify", "chain/service", "node/api", "node", "sync", "sync/api", "block-relayer", "miner", "node", "network-p2p", "network-p2p/types", "network-p2p/peerset", "network", "network/api", "network-rpc", "network-rpc/derive", "network-rpc/core", "network-rpc/api", "account/api", "account", "account/service", "rpc/api", "rpc/middleware", "rpc/client", "rpc/server", "vm/types", "vm/vm-runtime", "vm/natives", "vm/stdlib", "vm/compiler", "vm/move-prover", "vm/transaction-builder", "vm/transaction-builder-generator", "vm/move-coverage", "vm/resource-viewer", "vm/dev", "vm/move-explain", "vm/move-package-manager", "vm/vm-status-translator", "vm/starcoin-transactional-test-harness", "abi/types", "abi/decoder", "abi/resolver", "test-helper", "cmd/starcoin", "cmd/faucet", "cmd/tx-factory", "cmd/miner_client", "cmd/generator", "dataformat-generator", "cmd/indexer", "cmd/resource-exporter", "cmd/merkle-generator", "cmd/peer-watcher", "cmd/airdrop", "cmd/replay", "cmd/genesis-nft-miner", "stratum", "cmd/miner_client/api", "cmd/db-exporter",]
```
Comment (nkysg) [inline Cargo.toml:4]: 同上

Comment (welbon) [reply Cargo.toml]: fixed

File: Cargo.toml (hunk 1 above)
Comment (welbon) [reply Cargo.toml]: fixed

**Review by nkysg — APPROVED** [review]

**Pull request merged**